docs/_posts/2018-01-15-jekyll-3-7-1-released.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Jekyll 3.7.1 Released"
+date: 2018-01-15 11:21:00 +0530
+author: ashmaroli
+version: 3.7.1
+categories: [release]
+---
+
+Close on the heels of shipping 3.7.0, we were informed of a couple of
+regressions due to the changes made in that release. In due time, Team Jekyll
+set out to address those issues as early as possible.
+
+Days later, here we're announcing 3.7.1 that aims to set things in order. The
```

Comment:
unneccessary use of `here`

Reply:
also: can you replace `set things in order` with `set things right`? i think it's a more common idiom

Reply:
fair enough!